client/layout/masterbar/logged-out.jsx

```diff
@@ -11,6 +11,14 @@ import { localize } from 'i18n-calypso';
 import Item from './item';
 import config from 'config';
 
+function getLoginUrl() {
+	let loginUrl = config( 'login_url' );
+	if ( typeof window !== 'undefined' ) {
+		loginUrl += `?redirect_to=${ encodeURIComponent( window.location.href ) }`;
```

Comment:
True, but I don't think it improves the way the code flows - the idea is that `loginUrl` starts with some value and if we have `window`, we add to it and then return the `loginUrl` value either way. Returning early seems unnecessary in this case, so I'll leave it as-is if you don't mind.